langchain/src/tools/index.ts

```diff
@@ -47,3 +47,4 @@ export {
   formatToOpenAIFunction,
   formatToOpenAITool,
 } from "./convert_to_openai.js";
+export { JiraAPIWrapper, JiraAction } from "./jira.js";
```

Comment:
See the above comment on toolkits

You'll also need to prefix types with `type` when exporting to avoid build issues

Reply:
I have done the following:
* Added the following line to `create-entrypoints.js` under the `tools` key: `"tools/jira": "tools/jira,"`.
  * No additional prefixes for types needed to be added to the line `export { JiraAPIWrapper, JiraAction } from "./jira.js";` since only classes are exported.
  * Just as in the comment above, imports need to be adjusted accordingly.
  
Let me know if I need to make additional changes!